components/webdav/src/main/java/org/trellisldp/webdav/TrellisWebDAV.java

```diff
@@ -142,6 +142,13 @@ public TrellisWebDAV(final ServiceBundler services) {
         this(services, getConfig());
     }
 
+    /**
+     * Used only to support CDI.
+     */
+    TrellisWebDAV() {
+        this(null);
```

Comment:
In this case, is the `ServiceBundler` injected through some other means? Or is this merely to have a no-arg ctor available?

Reply:
The latter. It is frankly ugly, but it's CDI. There are impl-specific workarounds, but I don't want to do that, do you?

Briefly, because of the scoping of the use of this guy (and the others) in JAX-RS, CDI wants to use a proxy for the bean. It can use the `@Inject`-annotated fielded ctor for the impl of that proxy, but in order to build the proxy itself, it needs the no-arg ctor.

Reply:
Got it. Then this sounds good. I think there's a pesky checkstyle issue on line 76 of the TrellisWebDAVRequestFilter class, but otherwise LGTM.

Reply:
I'll check it now.